### PR TITLE
Create Foods.md

### PR DESCRIPTION
**Pull request by rizel-botany**

### CODE REVIEW

File: Foods.md [hunk 1]
```diff
@@ -0,0 +1 @@
+-[ ] Tofu
```
Comment (bailey-botany) [inline Foods.md:1]: Need to list all the types of tofu
- `teriyaki tofu`
- `jerk tofu`
- buffalo tofu

**Review by bailey-botany — CHANGES_REQUESTED** [review]

File: Annoying.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+legal
+
+aid
+
+helpful
+
+moe
+
+tyrone
+
```
[the remark this answers is not in the document]
Comment (rizel-botany) [reply Annoying.md:10]: game @bailey-botany

File: Annoying.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+legal
+
+aid
+
+helpful
+
```
[the remark this answers is not in the document]
Comment (rizel-botany) [reply Annoying.md:6]: stuck

File: Annoying.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+legal
+
+aid
+
```
[the remark this answers is not in the document]
Comment (rizel-botany) [reply Annoying.md:4]: point @bailey-botany

**Pull request closed without merging**